src/System.Management.Automation/FormatAndOutput/common/Utilities/MshObjectUtil.cs

```diff
@@ -290,17 +291,20 @@ internal static string SmartToString(PSObject so, PSPropertyExpressionFactory ex
                 // take care of the case there is no base object
                 return so.ToString();
             }
-            catch (ExtendedTypeSystemException e)
+            catch (Exception e)
             {
-                // NOTE: we catch all the exceptions, since we do not know
-                // what the underlying object access would throw
-                if (formatErrorObject != null)
+                if (e is ExtendedTypeSystemException || e is InvalidOperationException)
                 {
-                    formatErrorObject.sourceObject = so;
-                    formatErrorObject.exception = e;
+                    if (formatErrorObject != null)
+                    {
+                        formatErrorObject.sourceObject = so;
+                        formatErrorObject.exception = e;
+                    }
+
+                    return string.Empty;
```

Comment:
So users will see empty line instead of list of lines?

Reply:
This is augmenting the existing behavior to handle an additional exception case.